firefox-ios/Client/Frontend/Fakespot/Views/FakespotMessageCardView.swift

```diff
@@ -153,28 +153,25 @@ final class FakespotMessageCardView: UIView, ThemeApplicable, Notifiable {
         label.setContentCompressionResistancePriority(.required, for: .vertical)
     }
 
-    private lazy var linkButton: LegacyResizableButton = .build { button in
+    private lazy var linkButton: LinkButton = .build { button in
         button.titleLabel?.font = DefaultDynamicFontHelper.preferredFont(
```

Comment:
The font that is already configured in the `LinkButton` should not be overwritten. The font size of the button should be configured using the `LinkButtonViewModel`.

Reply:
Could you please remove the setting of the font as well? As we are using a standard component we should use the font of specified by the component.